Documentation/index.md

```diff
@@ -1,17 +1,88 @@
-Welcome to the MonoGame game library documentation hub.
+# Welcome to the MonoGame game library documentation hub.
 
 This area of the site contains the documentation on the API of MonoGame as well as how to use it to create great games.
-Note that this is a work in progress so there will be gaps in the documentation coverage. 
+Note that this is a work in progress so there will be gaps in the documentation coverage.
 
 If you cannot find what you need here you can also look at the [Microsoft XNA documentation](https://msdn.microsoft.com/en-us/library/bb200104.aspx). MonoGame is API compatible
-with [XNA](https://msdn.microsoft.com/en-us/library/bb203940.aspx) even down to the namespaces. So usually what works for XNA will work for MonoGame too. 
+with [XNA](https://msdn.microsoft.com/en-us/library/bb203940.aspx) even down to the namespaces. So usually what works for XNA will work for MonoGame too.
 
-Note that this documentation hub is built from the source code on every commit to the development branch. As such 
+> Note that this documentation hub is built from the source code on every commit to the development branch. As such
 it applies to the development builds available on the [Downloads](http://www.monogame.net/downloads) page. This may include new features which may
-not be available in the current stable release 
+not be available in the current stable release.
 
-### We Need Your Help!
+---
 
-Truly great open source projects require high quality documentation.  This is call for volunteers to help us make the MonoGame documentation truly great.  If you can help write tutorials, guides, code snippets, reference docs, video walkthroughs or just any improvement to our current documentation we could use your help!
+## Site Index
+
+The documentation site is made up of several major parts which are available as links at the top of the page.  If you are new to MonoGame, we highly recommend walking through the **[Getting Started](articles/getting_started/getting_started)** section to Install MonoGame and build your first game.
+
+### [Articles](/articles/introduction/introduction.md)
```

Comment:
Make the `Articles` and `API Reference` bullet points with descriptions like Microsoft docs do:

![image](https://user-images.githubusercontent.com/103522/88585558-d7690d00-d018-11ea-82b5-e4cfeff65511.png)

Reply:
Great idea @tomspilman 
Will also pull the text from the original site for prosperity (as shown above)